### DISCUSSION TITLE
Logo for the app

### DISCUSSION THREAD

**Discussion by med4kat** [Nerds-Who-Code/Mental-Health-Tracker#7] [General]
Can you think of the logo or draw the logo and attach it here for discussion?

**Comment by xintamosaik** [comment]
I like round-ish logos. I am not a good artist.

**Reply by med4kat** [reply]
Any kind of draft would be helpful. It should not be perfect.

**Comment by nicolecarman** [comment]
I can try my hand at a logo, but we need to come up with a name before we do this step.

**Reply by med4kat** [reply]
i think u r right 
i'll put high priority for the name qn
